CrewChiefV4/Audio/PlaybackModerator.cs

```diff
@@ -38,6 +59,72 @@ public static class PlaybackModerator
 
         public static int lastBlockedMessageId = -1;
 
+        private static Verbosity verbosity = Verbosity.FULL;
+        private static Dictionary<String, MessageQueueCounter> queuedMessageCounters = new Dictionary<string,MessageQueueCounter>();
```

Comment:
Is this for debugging, statistics?

Reply:
dunno yet, it might be removed if i can't find a use for it. Seemed like a good idea at the time